notebooks/supply_chain/example_utils.py

```diff
@@ -0,0 +1,98 @@
+from typing import List
+
+import numpy as np
+
+from maro.simulator import Env
+
+
+def get_facility_name2id_mapping(env: Env) -> dict:
+    facility_name2id = {}
+    for facility_id, facility_info in env.summary["node_mapping"]["facilities"].items():
+        facility_name = facility_info["name"]
+        facility_name2id[facility_name] = facility_id
+    return facility_name2id
+
+
+def get_unit_name2id_mapping(env: Env) -> dict:
+    unit_name2id = {}
+    for facility_id, facility_info in env.summary["node_mapping"]["facilities"].items():
+        facility_name = facility_info["name"]
+        for unit_name, unit_info in facility_info['units'].items():
+            if unit_info is None:
+                continue
+            if unit_name == "products":
+                for sku_id, sku_info in unit_info.items():
+                    unit_name2id[f"{facility_name}__{unit_name}{sku_id}"] = sku_info["id"]
```

Comment:
Is the pattern of the `name` defined here by yourself and used for the notebook only?

Reply:
For the notebook only.